annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -74,29 +81,52 @@ fun main(argv: Array<String>) {
     val repoToPushTo = params.getString("repoToPushTo")
     val localWorkingCopyDirectory = File(params.getString("localWorkingCopyDirectory"))
 
+    val exportedAnnotationRoot = params.getCreatableDirectory("exportedAnnotationRoot")
+
+    // Build params for exporting the annotations
     val exportAnnotationsParamsBuilder = Parameters.builder()
     exportAnnotationsParamsBuilder.set("inceptionUrl", params.getString("inceptionUrl"))
     exportAnnotationsParamsBuilder.set("inceptionUsername", params.getString("inceptionUsername"))
     exportAnnotationsParamsBuilder.set("inceptionPassword", params.getString("inceptionPassword"))
-    exportAnnotationsParamsBuilder.set("exportedAnnotationRoot","$localWorkingCopyDirectory" + params.getString("exportedAnnotationRoot"))
+    exportAnnotationsParamsBuilder.set("exportedAnnotationRoot", "$exportedAnnotationRoot")
```

Comment:
No need for the string interpolation. :-)

```suggestion
    exportAnnotationsParamsBuilder.set("exportedAnnotationRoot", exportedAnnotationRoot.getAbsolutePath())
```